NitroxClient/GameLogic/Spawning/WorldEntities/ReefbackWorldEntitySpawner.cs

```diff
@@ -33,7 +33,8 @@ public IEnumerator SpawnAsync(WorldEntity entity, Optional<GameObject> parent, E
             }
 
             life.initialized = true;
-            life.SpawnPlants();
+            // TODO: Choose whether or not to spawn plants
+            // life.SpawnPlants();
```

Comment:
Curious, what is the reasoning for why we would want to choose this? Is there a reason this is commented out?

Reply:
It just means that I'm postponing this for a further PR and haven't thought about reefbacks at all.